src/theme/components/image.ts

```diff
@@ -85,6 +85,7 @@ const CustomImage = {
     },
     figcaption: {
       fontStyle: "italic",
+      fontSize: "text.tag",
```

Comment:
Why are we switching which element the fontSize is added to?

Reply:
The element where `figcaption` is applied is the parent of the element where `captionWrappers` is applied. So it covers both "captionWrappers".

Reply:
Ah, okay. Thank you!